cmd/rancher_release/label_issues.go

```diff
@@ -0,0 +1,134 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/v39/github"
+	"github.com/rancher/ecm-distro-tools/release/semver"
+
```

Comment:
Remove blank line and  reformat imports.

Reply:
Formatted